report.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do you need this file?

Reply:
No, i'm not sure why it ended up in there...I'll remove it